pkg/preparer/orchestrate_test.go

```diff
@@ -82,7 +84,9 @@ func TestPreparerLaunchesPodsThatHaveDifferentSHAs(t *testing.T) {
 		currentManifest: existing,
 	}
 	newManifest := testManifest(t)
-	success := installAndLaunchPod(newManifest, testPod, logging.DefaultLogger)
+
+	var p *Preparer
```

Comment:
How doesn't this cause a panic? Declaring the pointer without initializing it should result in a dereferencing error or am I missing something